scipy/stats/_binned_statistic.py

```diff
@@ -650,6 +643,27 @@ def binned_statistic_dd(sample, values, statistic='mean',
     return BinnedStatisticddResult(result, edges, binnumbers)
 
 
+def _calc_binned_statistic(Vdim, binnumbers, result, values, stat_func):
+    unique_bin_numbers = np.unique(binnumbers)
+    for vv in builtins.range(Vdim):
+        bin_map = _create_bin_map(binnumbers, unique_bin_numbers, values, vv)
+        for i in unique_bin_numbers:
+            result[vv, i] = stat_func(bin_map[i])
+
+
+def _create_bin_map(binnumbers, unique_bin_numbers, values, vv):
+    """ Create bin map
+    key: bin number
+    value: list of binned data
+    """
+    bin_map = {}
```

Comment:
Were trying to use a `dict` ? If so, use `dict()` IIRC is b/c parsers conflate empty dicts w/sets.

Reply:
Oh. that's right. I fixed it.